Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml

```diff
@@ -151,10 +151,10 @@ args:
 comment: Create a phishing classifier using machine learning technique, based on email
   content.
 commonfields:
-  id: DBotBuildPhishingClassifier
```

Comment:
i think you mistake with the ID please change

Reply:
your'e right, will fix this, thanks